src/Pages/yewonchoi/Components/Feed/Comment/Comment.js

```diff
@@ -0,0 +1,94 @@
+import React, { Fragment } from "react";
+import "./Comment.scss";
+
+class Feed extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      inputValue: "",
+      comments: [],
+      commentbtn: false,
+    };
+  }
+
+  handleInput = (event) => {
+    this.setState({
+      inputValue: event.target.value,
+      commentbtn: event.target.value.length >= 1 ? true : false,
+    });
+  };
+
+  handleCommentInputKey = (event) => {
+    if (event.keyCode === 13 && this.state.inputValue.length >= 1) {
```

Comment:
length가 1보다 크다는 건 존재하는지를 묻고 있기 떄문에 아래와 같이 줄일 수 있겠네요!
```suggestion
    if (event.keyCode === 13 && this.state.inputValue) {
```